org.qutebrowser.qutebrowser.json

```diff
@@ -0,0 +1,217 @@
+{
+    "app-id": "org.qutebrowser.qutebrowser",
+    "branch": "stable",
+    "runtime": "org.freedesktop.Platform",
```

Comment:
I'm a bit confused, does this even work? This runtime doesn't have Qt.

Reply:
The whl of PyQt even ships Qt and all its dependencies.

Reply:
That sounds terrifying and not remotely supportable.